flink-core/src/main/java/org/apache/flink/configuration/RestartBackoffTimeStrategyOptions.java

```diff
@@ -54,4 +54,22 @@
 		.key("restart-backoff-time-strategy.failure-rate.backoff-time")
 		.defaultValue("1 min")
 		.withDescription("Backoff time between two consecutive restart attempts.");
+
+	/**
+	 * Maximum number of attempts the fixed delay restart strategy will try before failing a job.
+	 */
+	@PublicEvolving
+	public static final ConfigOption<Integer> RESTART_BACKOFF_TIME_STRATEGY_FIXED_DELAY_ATTEMPTS = ConfigOptions
+		.key("restart-backoff-time-strategy.fixed-delay.attempts")
+		.defaultValue(Integer.MAX_VALUE)
+		.withDescription("Maximum number of attempts the fixed delay restart strategy will try before failing a job.");
+
+	/**
+	 * Backoff time between two consecutive restart attempts in FixedDelayRestartBackoffTimeStrategy.
+	 */
+	@PublicEvolving
+	public static final ConfigOption<Long> RESTART_BACKOFF_TIME_STRATEGY_FIXED_DELAY_BACKOFF_TIME = ConfigOptions
+		.key("restart-backoff-time-strategy.fixed-delay.backoff-time")
+		.defaultValue(10_000L)
```

Comment:
Let's set the default value to `0`. With queued scheduling there is no longer a strict need for a delay.

Reply:
OK